packages/shared-secret/shared-secret.0.1/descr

```diff
@@ -0,0 +1 @@
+Abstract (encapsulated) messages or hidden (semi-deterministic) exceptions using OCaml's module system.
```

Comment:
This description is a bit long and the longer summary on subsequent lines is missing. It would be nice (but not necessary) to shorten this line and write a little more on later lines.